regression-test/suites/inverted_index_p0/test_array_contains_with_inverted_index.groovy

```diff
@@ -76,4 +76,18 @@ suite("test_array_contains_with_inverted_index"){
     order_qt_sql """ select * from tai where !array_contains(inventors, 's') and apply_date = '2019-01-01' order by id; """
     order_qt_sql """ select * from tai where !array_contains(inventors, 's') or apply_date = '2017-01-01' order by id; """
     order_qt_sql """ select * from tai where (array_contains(inventors, 's') and apply_date = '2017-01-01') or apply_date = '2019-01-01' order by id; """
+
+    // test arrays_overlap with inverted index
+    // now if we use inverted index we will not eval exprs
+    order_qt_sql """ select * from tai where arrays_overlap(inventors, array('s', 't')) order by id; """
+    order_qt_sql """ select * from tai where arrays_overlap(inventors, array('s', 't')) and apply_date = '2017-01-01' order by id; """
+    order_qt_sql """ select * from tai where arrays_overlap(inventors, array('s', 't')) and apply_date = '2019-01-01' order by id; """
+    order_qt_sql """ select * from tai where arrays_overlap(inventors, array('s', 't')) or apply_date = '2017-01-01' order by id; """
+    order_qt_sql """ select * from tai where !arrays_overlap(inventors, array('s', 't')) order by id; """
+    order_qt_sql """ select * from tai where !arrays_overlap(inventors, array('s', 't')) and apply_date = '2017-01-01' order by id; """
+    order_qt_sql """ select * from tai where !arrays_overlap(inventors, array('s', 't')) and apply_date = '2019-01-01' order by id; """
+    order_qt_sql """ select * from tai where !arrays_overlap(inventors, array('s', 't')) or apply_date = '2017-01-01' order by id; """
+    order_qt_sql """ select * from tai where (arrays_overlap(inventors, array('s', 't')) and apply_date = '2017-01-01') or apply_date = '2019-01-01' order by id; """
+
```

Comment:
may be more test here, like null value etc.

Reply:
done!